lib/src/turtle/parsettl.dart

```diff
@@ -0,0 +1,116 @@
+import '../graph.dart';
```

Comment:
Why do you have two functions in this file called `parseTTL` and `parseTTLStr`?

Reply:
Removed.